### ISSUE TITLE
Promotional elements on Film page

### ISSUE THREAD

**Issue by 2amarkina** [blockframes/blockframes#6878]
**Context:**
On the Banner of the Film Page, the action zone below the organization is reserved to the direct interaction with different promotional elements. 

![Снимок экрана 2021-10-13 в 11.36.31.png](https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/111f18c5-be19-4265-9fa4-1c31e135e6b0) 

Nevertheless we need to review this representation and install the rule of how many elements to show / what type etc. 

For the moment we had to limit the number of interaction buttons which is shown in the desktop version, as if not in the mobile version they are not represented correctly. 

**Task details**
We need to think over the desktop + mobile logic of representations that would let a user interact with promotional elements smoothly with no Header breaking in mobile.

**Comment by stephanie-cascade8** [comment]
UI: 1 day

**Comment by 2amarkina** [comment]
Should be also taken into account with [this one](https://app.zenhub.com/workspaces/design---ux-60ab5f37bd1f990011c933a2/issues/blockframes/blockframes/7333)

**Comment by 2amarkina** [comment]
[Documentation proposition](https://www.notion.so/cascade8/WIP-Banner-Film-Page-WIP-b1464efef5bb46958cd84fc361630268)

**Comment by 2amarkina** [comment]
All the mobile propositions are reflected in documentation and [wireframes are here](https://www.figma.com/file/xc51c8BsgG94IaIxjUq5m5/Film-Header-Banner?node-id=0%3A1)

**Comment by 2amarkina** [comment]
See the updates of promotional elements accordingly to the Film Banner structure changes ![film banner.png](https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/d2e2fc45-b49b-4685-ba37-72266e6c5257)

For the mobile structure for the moment let's not show the pomo elements, but keep with the main CTAs

**Comment by stephanie-cascade8** [comment]
> [Documentation proposition](https://www.notion.so/cascade8/WIP-Banner-Film-Page-WIP-b1464efef5bb46958cd84fc361630268)

can we update it with the new layout for mobile?

**Comment by stephanie-cascade8** [comment]
[Figma mockups ](https://www.figma.com/file/7jYrZUTAXck7lp4FBqt9cX/Components?node-id=46%3A123)(component link with it's different variants)

**Comment by stephanie-cascade8** [comment]
Note for UI : keep last "Content" header versions for now to avoid misunderstandings

**Comment by stephanie-cascade8** [comment]
@2amarkina I added the last corrections we talked abt on friday. @cams-goussale I think you reviewed too soon. Now is the most updated version but I will also make François check tomorrow. 

[Here is the link](https://www.figma.com/file/7jYrZUTAXck7lp4FBqt9cX/Components?node-id=46%3A123)

Tomorrow with françois we will review the mobile version:
- the view more button and it's dropdown
- the mat-buttons with icons only
- the centered more button in the end of page. 
- mobile modal behaviour in general

On the desktop versions we will review: 
- new gradient on banner picture to improve reading
- can we add the buttons animations (request asking price & ask screening) on mat-buttons?
